internal/operators/ocs/validations.go

```diff
@@ -77,7 +77,7 @@ func (o *operator) computeResourcesAllNodes(cluster *models.Cluster, ocsClusterR
 		*/
 		if !compactMode {
 			if host.Role == models.HostRoleAutoAssign {
-				status = "All host roles must be assigned to enable OCS."
+				status = "For OCS, all host roles must be assigned to worker for Standard Mode."
```

Comment:
What if the user has 6 hosts with auto-assign mode? The user would consider changing all host roles to worker according to the message.

Reply:
We can reword it to, "For OCS, all host roles must be assigned to master or worker for Standard Mode." WDYT?